it/repositories/PlaybackRepositorySpec.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import java.time.LocalDate
+
+import models.UserAnswers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import org.mongodb.scala.bson.BsonDocument
+import org.scalatest.{BeforeAndAfterEach, OptionValues}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.libs.json.Json
+import uk.gov.hmrc.mongo.test.MongoSupport
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class PlaybackRepositorySpec extends AnyWordSpec with Matchers
+  with ScalaFutures with OptionValues with MongoSupport with MongoSuite with BeforeAndAfterEach {
+
+  override def beforeEach() = Await.result(repository.collection.deleteMany(BsonDocument()).toFuture(),Duration.Inf)
```

Comment:
`await(repository.collection.deleteMany(BsonDocument()).toFuture())`

Could just swap out for this and `import play.api.test.Helpers.{await, defaultAwaitTimeout}`